tests/workspace/test_workflows.py

```diff
@@ -226,13 +227,19 @@ def test_some_workflows_not_found(mock_airlock_reporter):
     }
 
 
-def test_update_cache_file(mock_airlock_reporter, freezer, cache_path):
+@patch("workspace.workflows.jobs.RepoWorkflowReporter.write_cache_to_file")
+def test_cache_creation(mock_write, mock_airlock_reporter, freezer):
+    mock_write.return_value = None  # Disable writing to file and test separately
     assert mock_airlock_reporter.cache == {}
     freezer.move_to("2023-09-30 09:00:08")
-    mock_airlock_reporter.get_latest_conclusions()
+    _ = mock_airlock_reporter.get_latest_conclusions()
```

Comment:
Why do you need to do this?

Reply:
Hmm, I remember making this change but not why. Have reverted it for now as I am unlikely to recall why even down the line.